lib/broccoli/default-packager.js

```diff
@@ -579,29 +579,14 @@ module.exports = class DefaultPackager {
         minifyCSS: this.minifyCSS.options,
       };
 
-      let stylesAndVendor, preprocessedStyles;
+      let stylesAndVendor = callAddonsPreprocessTreeHook(this.project, 'css', tree);
```

Comment:
Would be great to get this "raw input" into a `broccoli-debug`

Reply:
https://github.com/ember-cli/ember-cli/pull/8289/commits/197aabe2dab76e2dbe9774692862a865118e3568